src/sds_data_model/metadata.py

```diff
@@ -28,11 +28,14 @@
 MetadataType = TypeVar("MetadataType", bound="Metadata")
 
 
-def _get_xpath(xpath: Union[str, Iterable[str], Iterable[Iterable[str]]]) -> str:
-    """Construct an `XPath`_ query from a string, list of strings, or list of lists of strings.
+def _get_xpath(xpath: Union[str, Iterable[str],
+                            Iterable[Iterable[str]]]) -> str:
+    """Construct an `XPath`_ query from a string, list of strings,
+    or list of lists of strings.
```

Comment:
This line is too long, how about:

```suggestion
    """`XPath`_ query from a string, list of strings, or list of lists of strings.
```